spec/moonshot/plugins/asg_spec.rb

```diff
@@ -2,20 +2,32 @@
   let(:name) { 'cdb-worker-dev-jarmes-WorkerASG-Q7DYM7901RBY' }
```

Comment:
Can we follow the class file path here on the spec `spec/moonshot/plugins/rotate_asg_instances/asg_spec.rb `